ESCPOS_NET/Emitters/BaseCommandValues/Chars.cs

```diff
@@ -2,7 +2,8 @@
 {
     public static class Chars
     {
-        public static readonly byte StyleMode = 0x21;
+        public static readonly byte StyleMode = 0x21;/**ASCII Char: !*/
+        public static readonly byte SizeMode = StyleMode;
```

Comment:
if they are always the same, I don't see why use a new one just for SizeMode, let's keep using the StyleMode char also for the Size command